games-arcade/circuslinux/circuslinux-1.0.3-r1.ebuild

```
@@ -1,37 +1,36 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
-inherit desktop
+EAPI=8
+
+inherit desktop toolchain-funcs
 
 DESCRIPTION="clone of the Atari 2600 game \"Circus Atari\""
-SRC_URI="ftp://ftp.sonic.net/pub/users/nbs/unix/x/circus-linux/${P}.tar.gz"
+SRC_URI="ftp://ftp.tuxpaint.org/unix/x/circus-linux/src/${P}.tar.gz"
 HOMEPAGE="http://www.newbreedsoftware.com/circus-linux/"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~x86"
-IUSE=""
 
-DEPEND="media-libs/libsdl
+DEPEND="
+	media-libs/libsdl[sound,video]
 	media-libs/sdl-image[png]
 	media-libs/sdl-mixer[mod]"
 RDEPEND="${DEPEND}"
 
 src_prepare() {
 	default
-	sed -i \
-		-e "/^install-data-am/d" \
-		Makefile.in \
-		|| die
-	sed -i \
-		-e 's/\$(pkgdatadir)/$(DESTDIR)&/' \
-		data/Makefile.in \
-		|| die
+	tc-export CC
+	sed -i -e 's/-Wall -O2//' \
+		Makefile.in || die
+	sed -i -e 's/\$(pkgdatadir)/$(DESTDIR)&/' \
+		data/Makefile.in || die
 }
 
```

This seem to have working joystick support but disables it by default, I don't see much reason to add a IUSE for this (joystick on libsdl is cheap and default) but also no reason to keep it disabled.
```suggestion

src_configure() {
	tc-export CC
	econf --enable-joystick
}

```
If defining src_configure, makes sense to move tc-export here too.

---

Nice catch, didn't noticed it. Added your suggestion and changed the deps and also made a rev-bump :)